src/session/session_compact.c

```diff
@@ -207,37 +207,43 @@ __session_compact_check_timeout(WT_SESSION_IMPL *session)
 
 /*
  * __wt_session_compact_check_interrupted --
- *     Check if compaction has been interrupted.
+ *     Check if compaction has been interrupted. Foreground compaction can be interrupted through an
+ *     event handler while background compaction can be disabled at any time using the compact
+ *     session API.
```

Comment:
Minor nit: I would call it the "session compact API" or just "the compact API". The ordering of the words and having "session" after is awkward.

Reply:
Thanks for the suggestion, fixed in 1fd2b1a32bbed859155fa50725e31f8ac7e24663.